cns/doc/examples/metrics/README.md

```diff
@@ -0,0 +1,50 @@
+# Azure CNS metrics
+azure-cns exposes metrics via Prometheus on `:10092/metrics`
+
+## Scraping 
+Prometheus can be configured using these examples: 
+- a [podMonitor](podMonitor.yaml), if using promotheus-operator or kube-prometheus
+- manually via this equivalent [scrape_config](scrape_config.yaml)
+
+## Monitoring
+To view all available CNS metrics once Prometheus is correctly configured to scrape:
+```promql
+count ({job="kube-system/azure-cns"}) by (__name__)
+```
+
+CNS exposes standard Go and Prom metrics such as `go_goroutines`, `go_gc*`, `up`, and more.
+
+Metrics designed to be customer-facing are generally prefixed with `cx_` and can be listed similarly:
+```promql
+count ({__name__=~"cx.*",job="kube-system/azure-cns"}) by (__name__)
+```
+At time of writing, the following cx metrics are exposed:
+- cx_ipam_available_ips
```

Comment:
Lets add same metrics we are exposing in Azure Monitor Workbook

IPs Allocated from Subnet == ipam_total_ips
ReserverForFuture = ipam_available_ips
AssignedToPods = ipam_pod_Allocated_ips

Reply:
Rest are internal and I dont know if they are any actionable to customers